docs/learn-papers.md

```diff
@@ -0,0 +1,29 @@
+---
+id: learn-papers
+title: Polkadot Papers
+sidebar_label: Polkadot Papers
+---
+
+## Polkadot's White Paper
+
+Polkadot's original white paper is a technical summary around one possible direction of implementing
+the Polkadot network. This paper uses rationale and technical details to support why this direction
+is beneficial. This original white paper also explains how each core component in Polkadot works
+together to build this decentralized network. Polkadot's white paper can be found
+[here](https://polkadot.network/PolkaDotPaper.pdf).
+
+## Polkadot's Overview Paper
+
+Polkadot's overview paper is a visual, easy to read, and less technical introduction into its
+blockchain technology. This paper dives into the components of Polkadot but is understandable for
+both a non-technical and technical reader. For reference, the overview paper can be found
+[here](https://polkadot.network/Polkadot-lightpaper.pdf).
```

Comment:
So actually, the overview paper is not the same as the Light Paper.

The [overview paper](https://github.com/w3f/research/blob/master/docs/papers/OverviewPaper-V1.pdf) is an updated version of the white paper that describes the protocol in more technical terms. We should primarily suggest this paper to people who want to dig more into the protocol.

Reply:
Oh! Thank you for this, I'll update it. Would it also be helpful to add in the light paper as another section on this page?

Reply:
Yes, I would suggest to keep the section the same but rename to Polkadot's Light Paper and make a new section for the Overview paper.